convert2rhel/pkghandler.py

```diff
@@ -895,3 +894,51 @@ def compare_package_versions(version1, version2):
     evr2 = utils.string_to_version(version2)
 
     return rpm.labelCompare(evr1, evr2)
+
+
+def get_total_packages_to_update():
+    """This is a simple function that returns the total number of packages that
+    need to be updated in the system. It uses both yum/dnf depending on weather
+    they are installed on the system.
+    """
+    packages = []
+
+    if pkgmanager.TYPE == "yum":
+        packages = _get_packages_to_update_yum()
+    elif pkgmanager.TYPE == "dnf":
+        packages = _get_packages_to_update_dnf()
+
+    return len(packages)
+
+
+def _get_packages_to_update_yum():
+    """Query all the packages with yum that has an update pending on the system."""
+    from cli import YumBaseCli
+
+    base = YumBaseCli()
+
+    packages = base.returnPkgLists(["updates"], None)
+
+    print("PACKAGES: ", packages)
```

Comment:
It's not done yet. The print is just for debugging purposes, will remove it when it's done